### DISCUSSION TITLE
Pinch or pan with two fingers

### DISCUSSION THREAD

**Discussion by caspg** [software-mansion/react-native-gesture-handler#2853] [Q&A]
I have problem composing pinch and two fingers pan gestures. I'm using `Gesture.Race` which works great on IOS but not so good on Android.

It looks that sometimes two fingers pan cancels pinch. Is there a way to only activate pan if two fingers are moving in the same direction?

**Comment by Glazzes** [comment]
The ideal use case would be to pan with the pinch gesture directly instead of using the pan gesture for panning, it sound a bit crazy, but that's how I do it in my library https://github.com/Glazzes/react-native-zoom-toolkit in case you want to give it a look.
